pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/fraser/Barnepensjon.kt

```diff
@@ -0,0 +1,277 @@
+        override fun OutlineOnlyScope<LangBokmal, Unit>.template() {
+            title1 {
+                text(
+                    Language.Bokmal to "Meld fra om endringer",
+                )
+            }
+            paragraph {
+                text(
+                    Language.Bokmal to "Du må melde fra med en gang det skjer viktige endringer i barnets liv, som"
+                )
+                list {
+                    item { text(Language.Bokmal to "endringer av nåværende familie- eller omsorgsforhold") }
+                    item { text(Language.Bokmal to "flytting eller opphold i et annet land over tid") }
+                    item { text(Language.Bokmal to "varig opphold i institusjon") }
+                }
+                text(
+                    Language.Bokmal to "Du er ansvarlig for å holde deg orientert om bevegelser på kontoen for utbetaling av barnepensjon, " +
+                            "og du må straks melde fra om eventuelle feil til NAV. Er det utbetalt for mye barnepensjon fordi " +
+                            "NAV ikke har fått beskjed, må pengene vanligvis betales tilbake. "
+                )
+            }
+        }
+    }
+
+    object EndringAvKontonummer : OutlinePhrase<LangBokmal>() {
+        override fun OutlineOnlyScope<LangBokmal, Unit>.template() {
+            title1 {
+                text(
+                    Language.Bokmal to "Endring av kontonummer",
+                )
+            }
+            paragraph {
+                text(
+                    Language.Bokmal to "Ved endring av kontonummer for utbetaling av barnepensjon til barn under 18 år, " +
+                            "kan du som forelder sende melding via nav.no/skrivtiloss eller sende skjema for melding om nytt " +
+                            "kontonummer per post. Du må da legge ved kopi av gyldig legitimasjon."
+                )
+            }
+            paragraph {
+                text(
+                    Language.Bokmal to "Oppnevnt verge må melde om endring via post. Det må legges ved kopi av egen legitimasjon og vergefullmakt."
+                )
+            }
+            paragraph {
+                text(
+                    Language.Bokmal to "Du finner mer informasjon og lenke til riktig skjema på nav.no/kontonummer."
+                )
+            }
+        }
+    }
+
+    object SkattetrekkPaaBarnepensjon : OutlinePhrase<LangBokmal>() {
+        override fun OutlineOnlyScope<LangBokmal, Unit>.template() {
+            title1 {
+                text(
+                    Language.Bokmal to "Skattetrekk på barnepensjon",
+                )
+            }
+            paragraph {
+                text(
+                    Language.Bokmal to "Barnepensjon er skattepliktig, men vi trekker ikke skatt for inneværende år " +
+                            "uten å få beskjed om dette. Du kan lese mer om frivillig skattetrekk på nav.no/skattetrekk#skatt-pengestotte."
```

Comment:
protip. Bruk konstanter for URL-er. Ganske praktisk etterhvert som dere får flere maler.